src/Controller/PostController.php

```diff
@@ -294,7 +294,7 @@ public function addCommentAction($id, Request $request = null)
         }
 
         $form = $this->getCommentForm($post);
-        $form->submit($request);
+        $form->handleRequest($request);
 
         if ($form->isValid()) {
```

Comment:
can you add an additional check isSubmitted

Reply:
Added the check